tests/System/SystemTestCase.php

```diff
@@ -49,6 +49,7 @@ public function phpbench($command, $workingDir = '.')
     {
         chdir($this->getWorkingDir($workingDir));
         $bin = __DIR__ . '/../../bin/phpbench --verbose';
+        $bin = str_replace('/', DIRECTORY_SEPARATOR, $bin);
```

Comment:
I just fixed this one occurence because I wanted to know how/if you'd want to take care of it. I've seen a few more instances. Most unit tests are working though, it's not that bad.